src/components/Display/Display.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Display.css'
+
+const Display = props => (<div className={'Display'}>
+    <h2 className={'Result'}>{props.result}</h2>
+</div>)
+
+Display.propTypes = {
+    result: PropTypes.string,
+}
+
+Display.defaultProps = {
```

Comment:
Related to my comment on `color` and `wide` prop, let us avoid writing `defaultProps` if we're not making this particular prop a required one. Please see my comment in `Button` component for guidance on how to do this.